pkg/agent/sidecar/service/observer/observer.go

```diff
@@ -139,12 +150,22 @@ func (o *observer) PostStop(ctx context.Context) (err error) {
 		return nil
 	}
 
-	ticker := time.NewTicker(o.postStopTimeout / 5)
+	ticker := time.NewTicker(func() time.Duration {
+		if o.postStopTimeout/5 < 2*time.Second {
+			return o.postStopTimeout / 5
+		}
+		return 2 * time.Second
+	}())
```

Comment:
ticker starts early I think it's better to locate this block before L197's for loop

Reply:
fixed in 25d315b